.gitattributes

```diff
@@ -0,0 +1,30 @@
+# Set line endings for known text files
+*.bat        text eol=crlf
+*.crn        text eol=lf
+*.css        text eol=lf
+*.dtd        text eol=lf
+*.ent        text eol=lf
+*.gradle     text eol=lf
+*.groovy     text eol=lf
+*.html       text eol=lf
+*.java       text eol=lf
+*.js         text eol=lf
+*.json       text eol=lf
+*.jsp        text eol=lf
+*.less       text eol=lf
+*.md         text eol=lf
+*.MF         text eol=lf
+*.Module     text eol=lf
+*.pl         text eol=lf
+*.pom        text eol=lf
+*.properties text eol=lf
+*.sh         text eol=lf
+*.svg        text eol=lf
+*.tld        text eol=lf
+*.txt        text eol=lf
+*.wsdl       text eol=lf
+*.xjb        text eol=lf
+*.xml        text eol=lf
+*.xsd        text eol=lf
+*.xsl        text eol=lf
+*.yml        text eol=lf
```

Comment:
[`*.md5`](https://github.com/Jasig/uPortal/blob/master/uportal-portlets-overlay/CalendarPortlet/3rd-party/repository/org/osaf/caldav4j/0.7/caldav4j-0.7-javadoc.jar.md5) and [`*.sha1`](https://github.com/Jasig/uPortal/blob/master/uportal-portlets-overlay/CalendarPortlet/3rd-party/repository/org/osaf/caldav4j/0.7/caldav4j-0.7-javadoc.jar.sha1) are two other unusual filetypes that exist.